common/lib/xmodule/xmodule/course_module.py

```diff
@@ -982,6 +982,13 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    credits = String(
+        display_name=_("Course credits number"),
```

Comment:
Do we need to marks this a CME specific field. Perhaps the display name should be "Course credits number for CME"